ledger/light/proc/hot_data.go

```diff
@@ -156,9 +155,15 @@ func (p *HotObjects) notifyPending(
 		return
 	}
 
-	_, err := p.Dep.MessageBus.Send(ctx, &message.AbandonedRequestsNotification{
-		Object: objectID,
-	}, nil)
+	msg, err := payload.NewMessage(&payload.AbandonedRequestsNotification{
+		ObjectID: objectID,
+	})
+	if err != nil {
+		inslogger.FromContext(ctx).Error("failed to create reply")
+	}
+
+	p.Dep.Sender.Reply(ctx, p.meta, msg)
```

Comment:
Not correct. Hot data was sent to us by another light. Notification should be send to virtual for this object.

Reply:
Ok. 
```
target, err := p.Dep.Coordinator.VirtualExecutorForObject(ctx, objectID, flow.Pulse(ctx))
p.Dep.Sender.SendTarget(ctx, msg, *target)
```